src/NBitcoin/Network.cs

```diff
@@ -230,6 +230,102 @@ internal static Network Register(Network network)
             return network;
         }
 
+        /// <summary>
+        /// Mines a new genesis block, to use with a new network.
+        /// Typically, 3 such genesis blocks need to be created when bootstrapping a new coin: for Main, Test and Reg networks.
+        /// </summary>
+        /// <param name="consensusFactory">
+        /// The consensus factory used to create transactions and blocks. 
+        /// Use <see cref="PosConsensusFactory"/> for proof-of-stake based networks.
+        /// </param>
+        /// <param name="pszTimestamp">
+        /// Traditionally a news headline from the day of the launch, but could be any string or link.
+        /// This will be inserted in the input coinbase transaction script.
```

Comment:
For sure there is some kind of size limit for this argument. It should be mentioned in the comment.

Reply:
So I had this before at 90 characters (I read that online) but the only limitation that I know of is the ScriptSig that's limited to 10,000 bytes.

Reply:
I think it is more like 200. I think there is consensus rule for that with some kind of limit around 200.

Reply:
Found it https://github.com/stratisproject/StratisBitcoinFullNode/blob/master/src/Stratis.Bitcoin.Features.Consensus/Rules/CommonRules/CheckPowTransactionRule.cs#L96

ScriptSig is max 100 so I'll set the limit under that.

Reply:
that is the one! i thought it was 200 :-) wroong!

Reply:
good catch tho 🐟